mod/robots_txt.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * return the default robots.txt
+ * @version 0.1.1
+ */
+
+/**
+ * Simple robots.txt
+ * @param App $a
+ * @return void
+ */
+function robots_txt_init(App $a) {
+
+	/** @var string[] globally disallowed url */
```

Comment:
please use `/*` or `//` for comments. If you youse `/**` it would be added to the doxygen documentation